src-plugins/itkFilters/itkFiltersBinaryCloseProcess_p.h

```diff
@@ -72,8 +72,8 @@ class itkFiltersBinaryCloseProcessPrivate : public itkMorphologicalFiltersProces
         QString newSeriesDescription = input->metadata ( medMetaDataKeys::SeriesDescription.key() );
 
         if (isRadiusInPixels)
-            newSeriesDescription += " Bin.Close filter\n("+ QString::number(radius[0])+", "+
-            QString::number(radius[1])+", "+ QString::number(radius[2])+" pixels)";
+            newSeriesDescription += " Bin.Close filter\n("+ QString::number(floor(radius[0]))+", "+
+            QString::number(floor(radius[1]))+", "+ QString::number(floor(radius[2]))+" pixels)";
```

Comment:
As Florent reopened the discussion, I can add a comment for the mm case: we should rather round (with an explicit floor(val+0.5) as Mathilde said) than truncate to be more precise and as users would expect (I actually don't really see where the pixel size intervenes in the computation from the mm radius, but I have pbs following the right file I may not be looking in the right place)